libs/design/src/molecules/sidebar/sidebar-viewport/sidebar-viewport.component.ts

```diff
@@ -133,7 +131,7 @@ export class DaffSidebarViewportComponent implements AfterContentChecked {
   private updateAnimationState() {
     this._animationState = {
       value: getAnimationState(
-        this.sidebars.reduce((acc: boolean, sidebar) => acc || isViewportContentShifted(sidebar.mode, sidebar.open), false),
+        this.sidebars.reduce((acc: boolean, sidebar) => acc || isViewportContentShifted(sidebar.mode, sidebar.open), false), DaffSidebarModeEnum.Over,
```

Comment:
This needs to be reviewed. I don't remember why we wrote it. It also needs a test.